admin_manual/installation/server_tuning.rst

```diff
@@ -146,9 +164,30 @@ To disable the revalidation completely:
 
   opcache.validate_timestamps = 0
 
-Any change to ``config.php`` will then require either restarting PHP, manually clearing the cache, or invalidating this particular script.
+Any Server/app upgrades or changes to ``config.php`` will then require restarting PHP (or otherwise manually clearing the cache or invalidating this particular script).
+
+.. warning::
+   To avoid false reports, if your environment isn't using the PHP default revalidation values, please do not report bugs/odd behavior after upgrading Nextcloud or Nextcloud apps until after you've 
+   restarted mod_php/fpm (to confirm they are not simply caused by local revalidation configuration).
+
+Sizing
+^^^^^^
+
+If any cache size limit is reached by more than 90%, the admin panel will show a related warning and suggested changes.
+
+For more details check out the `official PHP documentation <https://php.net/manual/en/opcache.configuration.php>`_. To monitor OPcache usage, clear individual or all cache entries, `opcache-gui <https://github.com/amnuts/opcache-gui>`_ can be used.
+
+Comments
+^^^^^^^^
+
+Nextcloud strictly requires code comments to be preserved in opcode, which is the default. But in case PHP settings are changed on your system, you may need set the following:
+
+.. code:: ini
+
+  opcache.save_comments = 1
 
-For more details check out the `official documentation <https://php.net/manual/en/opcache.configuration.php>`_. To monitor OPcache usage, clear individual or all cache entries, `opcache-gui <https://github.com/amnuts/opcache-gui>`_ can be used.
+JIT
+^^^
```

Comment:
If we touch this already, shall we update the JIT info as well?
- It works only on `x86_64`, not on ARM or any other architecture.
- `128M` size is completely oversized, and occupies the complete default overall OPcache size of 128M. I didn't see any instance with uses more than 1 MiB. JIT sizes in all cases I maintain are set to 2M, to have some buffer. Any recommendation or even just example of >8 MiB in the context of Nextcloud is problematic, triggering another warning due to filled overall OPcache, leading to unnecessarily occupied system memory when addressing this. Since I use `tracing` (=`on`=1254) instead of 1255, I just verified that the latter does not change anything about this. It is moreless precisely 1 MiB of JIT buffer size used. However, using a call graph instead of caching the whole script sounds more efficient, and some tests have shown that JIT is likely using more memory than what is defined via `opcache.jit_buffer_size`: The overall OPcache usage raises as well, whether intended or a bug, a little shady at least 😄. However, since it is the PHP default and recommendation (only disabled by Debian PHP package configs) I would use `opcache.jit = on` as example: https://www.php.net/manual/en/opcache.configuration.php#ini.opcache.jit
This caused already issues in both: Nextcloud docker and Nextcloud AIO appliances, and raised a question/request to align the docs with these findings: https://github.com/nextcloud/docker/pull/2185
In both cases, reducing JIT to `opcache.jit_buffer_size=8M` solves all admin panel warnings.
But of course it makes sense to have some others checking their actual JIT buffer usage (via [opcache-gui](https://github.com/amnuts/opcache-gui) or other ways), just in case my Debian PHP-FPM instances are somehow special cases, regarding JIT buffer usage 😉.

Reply:
Can be done in a dedicated PR.